components/Buttons.js

```diff
@@ -10,51 +10,63 @@ export const SmallButton = styled.button`
     color: darkgrey;
     background-color: #555555;
   }
+  &.back-office {
+    background-color: rgb(0, 0, 0, 0.9);
```

Comment:
Ich würde versuchen alles  in variablen anzulegen..

Reply:
done!